cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -366,7 +366,7 @@ if (NOT ARROW_FOUND)
     -DARROW_BUILD_TESTS=OFF)
 
   if ("$ENV{PARQUET_ARROW_VERSION}" STREQUAL "")
-    set(ARROW_VERSION "8309556c7d2b0e14df1422baa574cf2de8c1bd3b")
```

Comment:
Do we need more than 0.7.1 here?

If yes, I would like to make a parquet-cpp 1.3.1 release otherwise we can wait with this PR to be merged.

Reply:
Yes, we should probably release 1.3.1 without this and then focus on getting Arrow 0.8.0 out by the end of the month. There is also https://issues.apache.org/jira/browse/PARQUET-1122 -- I am not sure if I want to block 1.3.1 over this but we will need to take care of that soon

Reply:
This PR needs some very recent fixes in Arrow, so yes it needs more than 0.7.1. I'm fine waiting for arrow 0.8.0 then parquet-cpp 1.4.0